arviz/plots/distplot.py

```diff
@@ -90,6 +90,9 @@ def plot_dist(
         Keywords passed to ax.pcolormesh. Ignored for 1D KDE.
     hist_kwargs : dict
         Keywords passed to the histogram.
+    is_circular : {True, False, "degrees"}, default False
+        Select input type {"radians","degrees"} for circular histogram or KDE plot. If True,
+        default input type is "radians".
```

Comment:
Sorry to bug again on this. Thanks a lot for updating the docstring according to this, really appreciated. The docstring is great already, I would only add radians to the set and make `False` the first element (given that they are not ordered). Below there is the reasoning behind this suggestion. 

I am also learning numpydoc conventions and recommendations and in addition, ArviZ documentation is currently a mixture of different styles and conventions so it's hard to find good models of numpydoc style. I am trying to get new/updated arguments to use best practices hoping that with time the whole library will end up updated. 

In this particular case, the comment is about numpydoc _recommendations_ and not _conventions_ so we should go with whatever makes the docs easier to understand to our users. Here is the transcript from [numpydoc docstring guide](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard)

> When a parameter can only assume one of a fixed set of values, those values can be listed in braces, with the default appearing first:
>
>     order : {'C', 'F', 'A'}
>         Description of `order`.

which says nothing about combining this with optional arguments, the [pandas docstring guide](https://pandas.pydata.org/pandas-docs/stable/development/contributing_docstring.html#section-3-parameters) is more detailed:

> In case where there are just a set of values allowed, list them in curly brackets and separated by commas (followed by a space). If the values are ordinal and they have an order, list them in this order. Otherwise, list the default value first, if there is one

They also use either the `..., optional` or `..., default XXX` pattern together with the set notation, the two options I mentioned in the previous comment. 

I am also trying to see how to improve our documentation on writing docs #meta so any suggestions about improvements (or directly PRs) to the [contributing guide](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md) or the [developer guide](https://arviz-devs.github.io/arviz/developer_guide.html) (still in a very early draft stage) are very welcome.